### PR TITLE
#1178 Laptop Price Predictor

### LINKED ISSUE #1178: Title: Proposal for a laptop price predictor model built using machine learning and deep learning techniques like ANN and RNN.
### Is your proposal related to a problem? Please describe.

This project proposes a model to predict laptop prices. It leverages the power of ensemble learning, combining:

- Linear Regression
- Decision Trees
- Random Forest
- XGBoost Regressor
- Artificial Neural Network (ANN)
- Recurrent Neural Network (RNN)
The model considers various features like brand, CPU, RAM, storage, screen size, and more. Data will be preprocessed for optimal training.

Evaluation metrics like Mean Squared Error and Mean Absolute Error will assess the model's accuracy in predicting actual prices.

This project benefits consumers by enabling informed purchase decisions and assists businesses in market analysis.

### Add any other context or screenshots about the proposal request here.

_No response_

### PR DESCRIPTION
**Pull request by Gaurav-576**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?
Yes

### Description
It consists of  laptop price predictor machine learning models using multiple different machine learning and deep learning algorithms and also analysing the different models to get the best model.

Fixes #1178 

### Checklist
- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

### CODE REVIEW

**Review by Kushal997-das — COMMENTED** [review]
@Gaurav-576, I noticed that you have opened the same project in another repository. Therefore, I am closing this one. I won't mention this here as other admins might mark the PR as spam. Unfortunately, I can't add this project to this repository. Thank you.
